### PR TITLE
Merge 'CSS' into branch master

### PR DESCRIPTION
**Pull request by abhishekp2917**

### CODE REVIEW

**Comment by abhishekp2903** [comment]
Add default font-size for all header tags

File: css/custom.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+button{
+  border : none;
+  outline : none;
+}
+
+p{
+  font-size : 1.5rem;
+  font-weight : 400;
+}
+
```
Comment (abhishekp2903) [inline css/custom.css:15]: Add all the headers similar to h1 here

**Review by abhishekp2903 — COMMENTED** [review]
Complete the required work and restart the pull request

File: css/custom.css (hunk 1 above)
Comment (abhishekp2917) [reply css/custom.css:15]: OK I will do as required

**Comment by abhishekp2917** [comment]
Currently I have deleted the css branch

**Comment by abhishekp2903** [comment]
Thats really thoughtfull

**Pull request closed without merging**